app/models/post.rb

```diff
@@ -2,4 +2,12 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User'
   has_many :comments
   has_many :likes
+
+  def update_post_counter
+    author.update(posts_counter: author.posts.count)
+  end
```

Comment:
#### SUGGESTION
```suggestion
  def update_post_counter
    author.update(posts_counter: author.posts.count)
  end
  
after_save :update_post_counter

  private

```

- [ ] Good job so far, you can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post_counter method every time a new post has been saved to database, then make the update_post_counter method private